src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonTestHelper.cs

```diff
@@ -398,7 +398,7 @@ public static byte[] ReaderLoop(int inpuDataLength, out int length, ref Utf8Json
                 ReadOnlySpan<byte> valueSpan = json.HasValueSequence ? json.ValueSequence.ToArray() : json.ValueSpan;
                 if (json.HasValueSequence)
                 {
-                    Assert.True(json.ValueSpan == default);
+                    Assert.True(json.ValueSpan.IsEmpty);
```

Comment:
I think these tests wants to really test for `default`. This build break should be fixed by pragma warning disable.

Reply:
Yeah that's my thinking too. I wanted to see if there are more breaks after these ones are fixed.

Reply:
Ah, the analyzer should be disabled for test projects, could you please add a row here:

https://github.com/dotnet/runtime/blob/b5b290facb653201da48f9d4877d851787e601dc/eng/CodeAnalysis.test.globalconfig#L677

```
# CA2265: Do not compare Span<T> to 'null' or 'default'
dotnet_diagnostic.CA2265.severity = none
```

Reply:
I see the analyzer updated reverted, this can be added later when analyzer updated